lib/repo-type-mappings/yocto-based-OS-image/versionist.conf.js

```diff
@@ -56,8 +61,14 @@ module.exports = {
   },
   incrementVersion: (currentVersion, incrementLevel) => {
     if (isESR(currentVersion)) {
-      return semver.inc(currentVersion, 'patch')
+      if (isValidSemver(currentVersion) != null) {
+        return semver.inc(currentVersion, 'patch')
+      } else {
+        const patchVersion = Number(currentVersion.split('.')[2])
+        return currentVersion.slice(0, currentVersion.length - 1) + (patchVersion + 1)
```

Comment:
Am I reading this right that it only updates the last digit? What if it's more than one?

Reply:
You're right, I copied it from the history without realizing it had been fixed in https://github.com/product-os/balena-versionist/pull/68.

I'll change it.

Reply:
Done